README.md

```diff
@@ -0,0 +1,59 @@
+lib-webtopay
+========
+
+The Checkout API (Payment Gateway API) allows for the collection of online payments with many payment methods. 
+The Checkout API is easy to integrate – simply use one of our methods and the checkout processes will be performed 
+automatically. The library can be used to check all the necessary security parameters of transferred and received data.
+More information can be found in documentation: https://developers.paysera.com/en/checkout/basic
+
+Requirements
+============
+
+You must have PHP version 5.0 or higher
+
+
+Installation
+============
+
+Easiest way to use library is to include merged all-in-one file.
+It is located in base libwebtopay directory, "WebToPay.php".
+In this case you only need this one file.
+Example:
+```php
+    <?php
+
+    require_once('WebToPay.php');
+
+    // Your code goes here
+    
+    ?>
+```
+Alternatively, you can use files in the "src" folder.
+Either set-up autoloader or include file "includes.php" in "src" directory.
+Example:
+```php
+    <?php
+
+    require_once('libwebtopay/src/includes.php');
+
+    // Your code goes here
+    
+    ?>
+```
+Testing
+=======
+
+    $ cd /path/to/libwebtopay
+    $ phpunit
+
+
+Demo
```

Comment:
As this is not part of the library, and could make troubles (security, auto-completion on non-included files and similar), I'd suggest to remove these at all. If we'd like to provide a demo, it's best to do that in a separate repository